DeAceroDLabs/marketplace-frontend
# Feature/dmp 528 Confirm step payment view

## What is done

[[DMP-528] Confirm step payment view](https://deacero.atlassian.net/browse/DMP-528)

## How is done

- Added the confirm step on  `Payment` component.

## How to test

1. Run the project via `npm start`
2. /carrito/pago
3. You should see the confirm step on payment view.
